subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/LintCheck.kt

```diff
@@ -14,19 +14,28 @@ import org.gradle.kotlin.dsl.invoke
 class LintCheck(context: String, name: String) : SuppressibleBuildStep(context, name),
     ImpactAnalysisAwareBuildStep by ImpactAnalysisAwareBuildStep.Impl() {
 
+    var slackChannelForAlerts = ""
```

Comment:
channels are configurable per `ci step`

Reply:
if we will learn how to mark issues between teams it is still valuable to have summary channels